src/plugins/aliases.ts

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
```

Comment:
This plugin allows us to add in webpack aliases. In this case I'm adding `@components/` that is an alias for `src/components/` and can be referenced in our mdx files.